spec/services/discovery_engine/quality/sample_query_set_spec.rb

```diff
@@ -95,8 +95,18 @@
 
     let(:month_label) { :last_month }
 
-    it "returns the fully qualified GCP name of the sample query set" do
-      expect(sample_query_set.name).to eq("[location]/sampleQuerySets/clickstream_2025-10")
+    context "when table id is clickstream" do
+      it "returns the fully qualified GCP name of the sample query set" do
+        expect(sample_query_set.name).to eq("[location]/sampleQuerySets/clickstream_2025-10")
+      end
+    end
+
+    context "when table id is binary" do
+      let(:table_id) { "binary" }
+
+      it "returns the fully qualified GCP name of the sample query set" do
```

Comment:
I'd be tempted to say we're probably okay not to test that separately if it works for the other one!